src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/Handler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.Data.SqlClient.SNI;
+using Microsoft.Data.SqlClientX.IO;
+
+
+namespace Microsoft.Data.SqlClientX.Handlers
+{
+    internal interface IHandler<TRequest> where TRequest : HandlerRequest
```

Comment:
Eventually these should be separate files, but to unblock for now, we can have them in one file

Reply:
Likewise, there's some style changes that I'd push for

Reply:
I can take care of all this now. Tell me what I am missing ?